config/locales/fr.yml

```diff
@@ -230,17 +216,66 @@ fr:
       q: "Our coaches"
       a_p1: "You can see all the coaches have helped out in our workshops on the"
       a_p2: ". They are ordered by number of attendances. This is our way of showing our appreciation to them, as without them, we wouldn't be able to exist."
-
   chapters:
     intro: "Hi and welcome to codebar! Check out our <a href='http://codebar.io/faq'>FAQ</a> page for any questions you might have, and feel free to email us at %{email} if you can't find the answer you're looking for. Otherwise, use the button below to start the sign-up process."
     sign_up: "to be invited to our workshops."
     no_incoming: "There are no workshops coming up."
     contact: "To contact the %{city} team, send an email to %{email}"
-
+  job:
+    expired: 'This job post has expired'
+    published_on: 'Published on'
+    expires_on: 'Closing date'
+  jobs:
+    title: Jobs
+    index:
+      description: There are %{count} jobs listed
+      none: There are no jobs available.
+  feedback_form:
+    title: 'Workshop feedback'
+    description: Your submission will be completely anonymous. We read every piece of feedback and use it to help us improve our workshops.
+    note: Please do not mention any names, but instead choose your coach from the dropdown. This enables us to address issues but also to anonymously share feedback with coaches. Member privacy is very important to us!
+    rating: Your overall rating of the workshop
+    coach: 'Who was your coach?'
+    select_coach: '- Select Coach -'
+    tutorial: 'What tutorial did you go through?'
+    select_tutorial: '- Select Tutorial -'
+    request: 'How did you find the workshop?'
```

Comment:
I feel we need to change the wording of this question as I often I see people reply with the word Google 😂

Reply:
Maybe we can approach that in a different PR as the intention of this is just to get in the translation files for all languages.